Makefile

```diff
@@ -269,11 +266,8 @@ deploy-grafana:
 uninstall-grafana:
 	@echo "\n$(YELLOW)Local cluster $(CYAN)$(CLUSTER_GSLB1)$(NC)"
 	@echo "\n$(YELLOW)uninstall grafana $(NC)"
-	@$(eval PREVIOUS_CONTEXT := $(shell kubectl config current-context))
-	kubectl config use-context k3d-$(CLUSTER_GSLB1)
-	-kubectl delete -f deploy/grafana/dashboard-cm.yaml
-	@kubectl config use-context $(PREVIOUS_CONTEXT)
-	helm uninstall grafana -n k8gb --kube-context=k3d-$(CLUSTER_GSLB1)
+	-kubectl --context k3d-$(CLUSTER_NAME)1 delete -f deploy/grafana/dashboard-cm.yaml -n k8gb
```

Comment:
@jkremser do we still need error suppression here?

Reply:
I think, it's good to have it there so that the target wont fail if the grafana is not installed, similarly as you run `kubectl apply ..` twice (idempotency)

Reply:
I understand, however I see few issues with this approach:
- We should use the same approach for all "uninstall" and "destroy" targets then in order to be consistent.
- This target is not part of CI, and for end-user this suppression, won't have any difference, it will be similar experience in the case of and error:
```
❯ make uninstall-grafana

Local cluster 

uninstall grafana 
kubectl --context k3d-test-gslb1 delete -f deploy/grafana/dashboard-cm.yaml -n k8gb
Error from server (NotFound): error when deleting "deploy/grafana/dashboard-cm.yaml": configmaps "podinfo-dashboard" not found
make: [uninstall-grafana] Error 1 (ignored)
helm uninstall grafana -n k8gb --kube-context=k3d-test-gslb1
Error: uninstall: Release not loaded: grafana: release: not found
make: *** [uninstall-grafana] Error 1
```
- As you can see from the log, you might need to suppress helm errors too in order to maintain the idempotency.
So either we have to suppress all these errors, or just make things simple and fail fast.

Reply:
Makes sense, I didn't realize that `helm uninstall` was failing if it's not there. So removing the `-` (it will be also similar as in the prometheus targets)
(amended)

Reply:
@jkremser did you submit these changes already? The last commit I see is 4 days old.

Reply:
yep, but to wrong remote ;) (https://github.com/k8gb-io/k8gb/tree/grafana-target) it should be ok now